### ISSUE TITLE
A11y: notification bar live region and focus/content order

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#1981]
## Describe the Bug

The notification bar (e.g. to save or update a login) is in an `<iframe>`, wrapped inside a `<div aria-live="polite" ...></div>`. The intention here is that when the bar appears, assistive technologies should announce the bar's appearance/content. However, this doesn't currently work. A live region needs to first be "primed" - the container with `aria-live` needs to be added to a document first - this primes the browser/AT to be on the lookout for content changes from that point on. Then, after a small delay, content can be injected into the container and will then be announced. Generating the lot in one go has no effect (barring a few occasions where by sheer luck the container is generated just at the right time in one frame/loop, and the content in the next).

Additionally, the container and iframe are injected at the very end of the page/DOM. as a result, the bar (despite appearing at the very top of the viewport) comes last in the overall focus order. A keyboard user has to navigate through the entire page first, before reaching the bar. It should ideally be injected right at the top / start of the page instead in source order.

## Steps To Reproduce

* use a screen reader (NVDA, VoiceOver, JAWS, ...)
* trigger a notification (for instance, by attempting to log in somewhere, but with a different/wrong password, which will trigger the update notification)
* navigate through the page using `Tab`

## Expected Result

* the appearance of the bar (at the very least, the message itself) should be announced by the screen reader
* the controls in the bar should be the first elements to receive keyboard focus on the page

## Actual Result

* screen readers don't announce the appearance of the bar
* the controls in the bar (and the actual content as well, when navigating using reading keys/browse mode in the screen reader) come right at the end of the page

## Environment

- Operating system: Windows 10
- Browser: Chrome 92
- Build Version (go to "Settings" → "About" in the app): 1.51.1

## Additional Context

For the live region, I'd recommend - rather than trying to set the outer `<div>` to have `aria-live` and delaying the `<iframe>`'s injection, to instead move the live region into the bar itself (the "change-text", "add-text") - set that container to `aria-live`, then inject the message after a few milliseconds to trigger the screen reader announcement.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**Comment by patrickhlauke** [comment]
Yes, this issue/bug is still present.

**Comment by dwbit** [comment]
Thanks @patrickhlauke, I've removed the label.

**Comment by jprusik** [comment]
Hey @patrickhlauke - we've released a new version of the notification bar recently which should address the a11y concerns here. Consequently, I'm going to close this issue, but if you continue to experience the issues described in the new UX, please leave a comment and I'll open this back up.
